src/views/CreateShareStructure.vue

```diff
@@ -1,18 +1,346 @@
+    this.showShareStructureForm = true
+  }
+
+  private addEditShareClass (shareStructure: ShareClassIF): void {
+    let newList: ShareClassIF[] = [...this.shareClasses]
+    // New Share Structue.
+    if (this.activeIndex === -1) {
+      newList.push(shareStructure)
+    } else {
+      // Edit Share Structure.
+      newList.splice(this.activeIndex, 1, shareStructure)
+    }
+    this.setShareClasses(newList)
+    this.setCreateShareStructureStepValidity(this.shareClasses.length > 0)
+    this.resetData()
+  }
+
+  private editSeries (index: number, seriesIndex: number): void {
+    this.activeIndex = seriesIndex
+    this.parentIndex = index
+    let newList: ShareClassIF[] = [...this.shareClasses]
+    this.currentShareStructure = { ...newList[this.parentIndex].series[this.activeIndex] }
+    this.addEditInProgress = true
+    this.showShareStructureForm = true
+  }
 
-@Component({})
-export default class CreateShareStructure extends Vue {}
+  private removeSeries (index: number, seriesIndex: number): void {
+    this.activeIndex = seriesIndex
+    this.parentIndex = index
+    let newList: ShareClassIF[] = [...this.shareClasses]
+    const parentShareClass = newList[this.parentIndex]
+    let series = [...parentShareClass.series]
+    series.splice(this.activeIndex, 1)
+    parentShareClass.series = series
+    this.setShareClasses(newList)
+    this.resetData()
+  }
+
+  private addEditShareSeries (shareSeries: ShareClassIF): void {
+    let newList: ShareClassIF[] = [...this.shareClasses]
+    const parentShareClass = newList[this.parentIndex]
+    let series = [...parentShareClass.series]
+    // New Share Structue.
+    if (this.activeIndex === -1) {
+      series.push(shareSeries)
+    } else {
+      // Edit Share Structure.
+      series.splice(this.activeIndex, 1, shareSeries)
+    }
+    parentShareClass.series = series
+    this.setShareClasses(newList)
+    this.resetData()
+  }
+
+  private removeShareClass (index: number): void {
+    let newList: ShareClassIF[] = [...this.shareClasses]
+    newList.splice(index, 1)
+    this.setShareClasses(newList)
+    this.setCreateShareStructureStepValidity(this.shareClasses.length > 0)
+    this.resetData()
+  }
+
+  private resetData (): void {
+    this.currentShareStructure = null
+    this.activeIndex = -1
+    this.addEditInProgress = false
+    this.showShareStructureForm = false
+    this.parentIndex = -1
+    this.nextId = -1
+  }
+
+  private get showErrors (): boolean {
+    return Boolean(this.$route.query.showErrors)
+  }
+}
 </script>
 
 <style lang="scss" scoped>
+@import '@/assets/styles/theme.scss';
+
+[class^="col"] {
+  padding-top: 0;
+  padding-bottom: 0;
+}
+
+.add-share-structure-container {
+  margin-top: 1rem;
+  margin-bottom:1rem;
+}
+
+ul {
+  padding-top: 0.5rem;
+  list-style: none;
+  margin-left: 0;
+  padding-left: 1rem
+}
+
+li {
+  padding-top:0.25rem
+}
+
+p{
+  padding-top: 0.5rem;
+}
+
+.btn-panel {
+  padding: 2rem 0 2rem 0;
```

Comment:
FYI, a shorthand for this is `2rem 0`.
https://developer.mozilla.org/en-US/docs/Web/CSS/Shorthand_properties

Reply:
Fixed